lib/rules/dot-location.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @fileoverview Validates newlines before and after dots
+ * @author Greg Cochard
+ * @copyright 2015 Greg Cochard
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function (context) {
+
+    var config = context.options[0],
+        // default to onObject if no preference is passed, alias obj to object
+        onObject = config === "object" || config === "obj" || !config;
```

Comment:
I don't think it's a good idea to have an alias. Object is not much longer then obj. Undocumented features = bad:-)